addons/resource.language.en_gb/resources/strings.po

```diff
@@ -19518,6 +19523,12 @@ msgctxt "#36179"
 msgid "When a file is scanned into the library it will display the metadata title instead of the file name."
 msgstr ""
 
+#. Description of setting with label #29995 "Allow movies in 'Disc n' subdirectories"
+#: system/settings/settings.xml
+msgctxt "#39198"
+msgid "When a 'Disc n' subdirectory is found a set is created and (Disc n) appended to the title of each entry in the set"
```

Comment:
Now that we support Disc, Disk and localized version according to Kodi language settings, you need to extend this I guess.